### PR TITLE
Bug 2051563: add popover for MCG clusters to depict no limit.

### PR DESCRIPTION
**Pull request by weirdwiz**
Screenshot from 2022-09-15 14-59-03

https://bugzilla.redhat.com/show_bug.cgi?id=2051563

### CODE REVIEW

File: packages/shared/dashboards/capacity-card/capacity-card.tsx [hunk 1]
```diff
@@ -203,6 +205,18 @@ const CapacityCardRow: React.FC<CapacityCardRowProps> = ({
       </GridItem>
       <GridItem span={3} key={`${data?.name}~value`}>
         {dataUnavailable ? '-' : value}
+        {isPercentage ? (
```
Comment (weirdwiz) [inline packages/shared/dashboards/capacity-card/capacity-card.tsx:208]: @bipuladh this is under the assumption that there are only two scenarios. Is this correct?

**Comment by weirdwiz** [comment]
PTAL @GowthamShanmugam @SanjalKatiyar @bipuladh

File: packages/shared/dashboards/capacity-card/capacity-card.tsx [hunk 2]
```diff
@@ -203,6 +205,18 @@ const CapacityCardRow: React.FC<CapacityCardRowProps> = ({
...
+          ''
+        ) : (
+          <Popover
+            position="right"
+            bodyContent="For standalone MCG clusters, there is no set limit."
```
Comment (SanjalKatiyar) [inline packages/shared/dashboards/capacity-card/capacity-card.tsx:213]:
```suggestion
            bodyContent=t('For standalone MCG clusters, there is no set limit.')
```

File: packages/shared/dashboards/capacity-card/capacity-card.tsx [hunk 3]
```diff
@@ -203,6 +205,18 @@ const CapacityCardRow: React.FC<CapacityCardRowProps> = ({
...
+          >
+            <Button variant="plain" className="details-item__popover-button">
+              No Limit
```
Comment (SanjalKatiyar) [inline packages/shared/dashboards/capacity-card/capacity-card.tsx:216]:
```suggestion
              {t('No Limit')}
```

File: packages/shared/dashboards/capacity-card/capacity-card.tsx [hunk 4]
```diff
@@ -143,7 +145,7 @@ const CapacityCardRow: React.FC<CapacityCardRowProps> = ({
       : 0;
   const value = isPercentage
     ? `${data?.usedValue?.string} / ${data?.totalValue?.string}`
-    : data?.usedValue?.string;
+    : data?.usedValue?.string + ' / ';
```
Comment (SanjalKatiyar) [inline packages/shared/dashboards/capacity-card/capacity-card.tsx:148]:
```suggestion
    : data?.usedValue?.string;
```

File: packages/shared/dashboards/capacity-card/capacity-card.tsx (hunk 1 above)
Comment (SanjalKatiyar) [inline packages/shared/dashboards/capacity-card/capacity-card.tsx:208]:
```suggestion
        {isPercentage && ( <Popover
```

File: packages/shared/dashboards/capacity-card/capacity-card.tsx [hunk 5]
```diff
@@ -203,6 +205,18 @@ const CapacityCardRow: React.FC<CapacityCardRowProps> = ({
       </GridItem>
       <GridItem span={3} key={`${data?.name}~value`}>
         {dataUnavailable ? '-' : value}
+        {isPercentage ? (
+          ''
+        ) : (
+          <Popover
+            position="right"
```
Comment (SanjalKatiyar) [inline packages/shared/dashboards/capacity-card/capacity-card.tsx:212]: can't we have it on auto ? else its position will be right on smaller screen sizes as well...

**Comment by SanjalKatiyar** [comment]
test this PR once on Full Deployment mode as well, just to make sure there is no regression !

File: packages/shared/dashboards/capacity-card/capacity-card.tsx (hunk 1 above)
Comment (SanjalKatiyar) [reply packages/shared/dashboards/capacity-card/capacity-card.tsx]: @bipuladh can there be any issue for external mode cluster (RHCS) ? do we have used/total there ?

Comment (bipuladh) [reply packages/shared/dashboards/capacity-card/capacity-card.tsx]: For External mode we do have used/total information.

File: packages/shared/dashboards/capacity-card/capacity-card.tsx (hunk 2 above)
Comment (bipuladh) [reply packages/shared/dashboards/capacity-card/capacity-card.tsx]: Lets ask UX team to review this once.

File: packages/shared/dashboards/capacity-card/capacity-card.tsx (hunk 5 above)
Comment (Chandan141996) [reply packages/shared/dashboards/capacity-card/capacity-card.tsx:211]: @weirdwiz @SanjalKatiyar For larger screen sizes I guess "right" should be the default value, for small screens "auto" should be used to self align the popover with the screen size

Comment (weirdwiz) [reply packages/shared/dashboards/capacity-card/capacity-card.tsx]: Having the value as "right" will automatically change the position if the screen size goes small. 
Attached screenshot below:
Screenshot from 2022-09-29 16-07-38
Screenshot from 2022-09-29 16-07-02

File: packages/shared/dashboards/capacity-card/capacity-card.tsx [hunk 6]
```diff
@@ -203,6 +206,21 @@ const CapacityCardRow: React.FC<CapacityCardRowProps> = ({
...
+          <span>
+            &nbsp; / &nbsp;
+            <Popover
+              position="right"
+              bodyContent={t(
+                'For standalone MCG clusters, there is no set limit.'
+              )}
+            >
+              <Button variant="plain" className="details-item__popover-button">
+                {t('No Limit')}
```
Comment (SanjalKatiyar) [inline packages/shared/dashboards/capacity-card/capacity-card.tsx:219]: why not directly this ??
```suggestion
                {t(' / No Limit')}
```

Comment (weirdwiz) [reply packages/shared/dashboards/capacity-card/capacity-card.tsx:219]: Adding it here adds the underline/hyperlink under the "/" too, which wasn't present in the UX design :sweat_smile: .

This is why I separated the "/"

**Pull request merged**